sky/client/cli/command.py

```diff
@@ -3969,7 +3969,7 @@ def _output() -> Generator[str, None, None]:
 
             list_accelerator_counts_result = sdk.stream_and_get(
                 sdk.list_accelerator_counts(
-                    gpus_only=True,
+                    gpus_only=not show_all,
```

Comment:
The changes in this file should be unnecessary. The two chips are available after the change in the above file with `gpus_only=True`.